src/nunjucks/moduk/components/phase-banner/__examples__/default.njk

```diff
@@ -0,0 +1,12 @@
+{%- from "moduk/components/phase-banner/macro.njk" import modukPhaseBanner -%}
+
+{{ modukPhaseBanner({
+  tag: {
+    text: "alpha"
+  },
+  classes: "custom-class",
+  attributes: {
+    "data-testid": "phase-banner"
+  },
+  html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+}) }}
```

Comment:
Here I wanted to test that the additional params are still passed to the element.